source/templates/layouts/_modal.twig

```diff
@@ -0,0 +1,27 @@
+{# How to use
+  {% extends "layouts/_modal.twig" %}
+
+  {% set modal = {
+    title: 'Modal title',
+    btnSubmit: 'Add project',
+    btnReset: 'Reset'
+  } %}
+
+  {% block modalContent %}This is modal content{% endblock %}
+#}
+
+<div class="overlay"></div>
```

Comment:
We will create this element with JS when we will display the modal on the page, remove it